templates/blocks/spreadsheet.twig

```diff
@@ -11,13 +11,9 @@
 					<tr data-order="{{ rowNumber }}">
 						{% for cell in row %}
 							{% if rowNumber == 0 %}
-								<th title="{{ __('Sort by') }}">
-									{{ cell|escape }}
-								</th>
+								<th title="{{ __('Sort by') }}">{{ cell|escape }}</th>
```

Comment:
Looks good! I think we should add the `planet4-blocks` domain here in the translator function.